agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -383,6 +384,11 @@ function install_k8s_agent {
         AGENT_NAMES="agent-slim agent-kmodule"
     fi
 
+	if [ ! -z "$IMAGE_PULL_SECRET" ]; then
+		sed -i.bak -e "s|#imagePullSecrets:|imagePullSecrets:|" $DAEMONSET_FILE
+		sed -i.bak -e "s|#- name: secret-name|- name: ${IMAGE_PULL_SECRET}|" $DAEMONSET_FILE
```

Comment:
I think you should be able to use a single command:
```suggestion
		sed -i.bak -e "s|#imagePullSecrets:|imagePullSecrets:|" \
			-e "s|#- name: secret-name|- name: ${IMAGE_PULL_SECRET}|" \
			$DAEMONSET_FILE
```